src/ptvsd/wrapper.py

```diff
@@ -541,11 +541,15 @@ def __init__(self, proc):
         self.exceptions = {}
         self.lock = threading.Lock()
 
+    def remove_exception_break(self, ex_type=None, exception='BaseException'):
```

Comment:
Might as well just make it `ex_type='python'` and skip the check inside, since strings are immutable.